File: VB/ChartsDemo.Wpf/DemoTesting/Fixtures.vb
Imports System
Imports System.Linq
Imports DevExpress.Xpf.DemoBase.DemoTesting

Namespace ChartsDemo.Tests
    Public Class ChartsCheckAllDemosFixture
        Inherits CheckAllDemosFixture

        Private skipMemoryLeaksCheckModules() As Type = { }

        Protected Overrides Function CheckMemoryLeaks(ByVal moduleType As Type) As Boolean
            Return Not skipMemoryLeaksCheckModules.Contains(moduleType)
        End Function
    End Class
End Namespace

File: CS/ChartsDemo.Wpf/DemoTesting/Fixtures.cs
using System;
using System.Linq;
using DevExpress.Xpf.DemoBase.DemoTesting;

namespace ChartsDemo.Tests {
    public class ChartsCheckAllDemosFixture : CheckAllDemosFixture {
        Type[] skipMemoryLeaksCheckModules = new Type[] { };

        protected override bool CheckMemoryLeaks(Type moduleType) {
            return !skipMemoryLeaksCheckModules.Contains(moduleType);
        }
    }
}
